taqueria-vscode-extension/src/lib/gui/SandboxesDataProvider.ts

```diff
@@ -208,10 +208,19 @@ export class SandboxesDataProvider extends TaqueriaDataProviderBase
 		if (!tzktBaseUrl) {
 			return [];
 		}
-		const response = await fetch(`${tzktBaseUrl}/v1/contracts/${element.parent.address}/entrypoints`);
+		const response = await fetch(
```

Comment:
Might be a good idea to wrap this in a try-catch to handle potential network errors

Reply:
Yes, agree, nice catch. We should probably take care of that for all calls to TzKt. Maybe during a future PR that is planned to improve TzKt calls for public nodes.